src/wbi/commands/goodbye.py

```diff
@@ -23,7 +23,6 @@ def add_args(parser):
 
 
 def main(args):
-
```

Comment:
I'm curious why these newlines have gone. Is this something you did consciously (which is fine), or was it done automatically through a `pre-commit` hook or something?

Reply:
We don't need to run these tools individually, because different versions of these tools will have different opinions, which is why none of these dependencies are specified in our `pyproject.toml`.

The fact that this file is passing the CI on the `main` branch means that the existing formatting is "correct". Can you check if reverting to the old version and just running `pre-commit run --all-files` changes the formatting in the same way? The versions of the tools that should be run on commit are hardcoded there, so they should behave identically on your laptop/mine/CI etc.

Reply:
The reason I ran the black command was because the CI was failing. The exact error case is shown in the screenshot below: 
<img width="870" alt="Screenshot 2023-12-01 at 2 35 39 PM" src="https://github.com/vineetbansal/wbi/assets/66816733/f5cfe174-36b4-4f52-871f-e2f8bdc6872a">
The pre-commit tests only passed after I formatted my files using black. I will revert it back again and see if running `pre-commit run --all-files` will change anything.

Reply:
The `pre-commit run --all-files` did not do anything to the new line but it did remove trailing whitespaces.

Reply:
ok - so that's probably why the CI was failing. Make sure you do a `pre-commit install` in the git root folder after activating the environment where you do `wbi` development so these checks run automatically. (as explained in the wiki).